bgpd/bgp_rpki.c

```diff
@@ -932,7 +932,7 @@ static int config_write(struct vty *vty)
 
 		vty_out(vty, "preference %hhu\n", cache->preference);
```

Comment:
Should this have a space added?

Reply:
No, this is a continuation of the `rpki cache ...` config line.